.github/workflows/static.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '8.0'
+          php-version: '7.4'
```

Comment:
psalm has a false positive that the function doesn't exist, because it can't pick up the polyfill.